android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/HomeActivity.java

```diff
@@ -17,89 +14,62 @@
 import android.view.View;
 import android.widget.Button;
 import android.widget.TextView;
-import com.liferay.apio.consumer.model.Thing;
-import com.liferay.mobile.android.service.Session;
 import com.liferay.mobile.formsscreenletdemo.R;
-import com.liferay.mobile.formsscreenletdemo.service.APIOFetchResourceService;
-import com.liferay.mobile.formsscreenletdemo.util.Constants;
-import com.liferay.mobile.formsscreenletdemo.util.DemoUtil;
-import com.liferay.mobile.formsscreenletdemo.util.ResourceType;
+import com.liferay.mobile.formsscreenletdemo.presenter.HomePresenter;
+import com.liferay.mobile.formsscreenletdemo.contract.HomeViewContract;
 import com.liferay.mobile.formsscreenletdemo.view.login.LoginActivity;
 import com.liferay.mobile.formsscreenletdemo.view.sessions.SpecialOffersActivity;
 import com.liferay.mobile.formsscreenletdemo.view.sessions.TakeCareListActivity;
 import com.liferay.mobile.formsscreenletdemo.view.sessions.BlogPostingsActivity;
-import com.liferay.mobile.push.Push;
 import com.liferay.mobile.screens.context.SessionContext;
-import com.liferay.mobile.screens.context.storage.CredentialsStorageBuilder;
-import com.liferay.mobile.screens.ddm.form.model.FormInstanceRecord;
-import com.liferay.mobile.screens.ddm.form.service.APIOFetchLatestDraftService;
 import com.liferay.mobile.screens.thingscreenlet.screens.ThingScreenlet;
-import com.liferay.mobile.screens.thingscreenlet.screens.views.Custom;
-import com.liferay.mobile.screens.util.AndroidUtil;
-import com.liferay.mobile.screens.util.LiferayLogger;
-import kotlin.Unit;
-import org.json.JSONException;
 
 /**
  * @author Luísa Lima
  * @author Victor Oliveira
  */
-public class HomeActivity extends AppCompatActivity {
+public class HomeActivity extends AppCompatActivity implements HomeViewContract.HomeView{
 
+	public ThingScreenlet userPortrait;
```

Comment:
Why this is public? This has to be private, remember the encapsulation has to be preserved.